README.md

```diff
@@ -1,84 +1,119 @@
-6) Run `./bin/pending_publisher` in background. This process is not strictly necessary but highly recommended. 
-It checks for possible errors publishing transactions on chain and try to resend it.
-7) Follow the [steps](#steps-to-write-the-private-key-in-the-vault) to write the private key in the vault
-8) Set up [required environment variables](#how-to-configure)
-
-
-## How to configure
-The different components must be configured with environment variables. We provide three env files samples with the required variables.
-Similar files are actually used by the [docker-compose](infrastructure/local/docker-compose.yml) file provided.
-
-### Steps to write the private key in the vault
-Just run:   
-```
-make private_key=xxx add-private-key
-```
-
-## How to test it
-1) Start the testing environment 
-``make up-test``
-2) Run tests
-``make tests`` to run test or ``make test-race`` to run tests with go test --race
-3) Run linter
-``make lint``
+This is a set of tools and APIs for issuers of zk-proof credentials, designed to be extensible. It allows an authenticated user to create schemas for issuing and managing credentials of identities. It also provides a web-based [frontend (UI)](ui/README.md) to manage issuer schemas, credentials and connections.
+
+## Installation
+
+There are two options for installing and running the server alongside the UI.
+
+### Option 1 - Using Docker only
+
+Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
+
+#### Requirements for Docker-only
+
+- [Docker Engine](https://docs.docker.com/engine/) 1.27+
+- Makefile toolchain
+- Unix-based operating system (e.g. Debian, Arch, Mac OS X)
+
+_NB: There is no compatibility with Windows environments at this time._
+
+#### Setup for Docker-only
+
+1. Copy `.env-api.sample` as `.env-api` and `.env-issuer.sample` as `.env-issuer`. Please see the [configuration](#configuration) section for more details.
+2. Run `make up`. This launches 3 containers with Postgres, Redis and Vault. Ignore the warnings about variables, since those are set up in the next step.
+3. **If you are on an Apple Silicon chip (e.g. M1/M2), run `make run-arm`**. Otherwise, run `make run`. This starts Docker containers for the issuer application.
+4. Follow the [steps](#adding-ethereum-private-key-to-the-vault) for adding an Ethereum private key to the Vault.
+5. Open <http://localhost:3001> in a browser (or whatever was set in the `[Server] URL` config entry). This shows an admin interface for documentation and credentials issuer setup.
+6. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#configuration) section for more details.
+7. _(Optional)_ Run `make-run-ui` (or `make-run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:80>. Its HTTP auth credentials are set in `.env-ui`.
```

Comment:
these commands are not correct:

`make run-ui` and `make run-arm-ui`

Reply:
Oops, thanks!

Reply:
The port is also incorrect: `http://localhost:8080/`

Reply:
> The port is also incorrect: http://localhost:8080/

This is fixed.

> I'd consider changing the command make run-arm-ui to make run-ui-arm

It's already `make run-ui-arm`.